client/my-sites/domains/domain-management/transfer/transfer-to-other-site/index.jsx

```diff
@@ -51,9 +51,12 @@ class TransferToOtherSite extends React.Component {
 	}
 
 	isSiteEligible = site => {
+		// check if the iste is Atomic from the site options
+		const isAtomic = get( site, 'options.is_automated_transfer', false );
```

Comment:
I'd probably move this to `connect()` as we do for `isDomainOnly: isDomainOnlySite( state, ownProps.selectedSite.ID )`.

Reply:
Actually, I thought on this at the first moment, but it isn't easy to do because what we need to check is site by site, if it's an Atomic one. The `ownProps.selectedSite.ID` is making reference to the current site.